### PR TITLE
`lxc exec` fixes

### PR DESCRIPTION
**Pull request by stgraber**
The previous fix for the SIGWINCH issue on non-interactive incorrectly
used "return" rather than "continue", which then caused the websocket to
terminate rather than for the signal to be ignored.

Signed-off-by: Stéphane Graber <stgraber@ubuntu.com>

### CODE REVIEW

**Comment by brauner** [comment]
jenkins: test this please

File: lxd/events/common.go [hunk 1]
```diff
@@ -45,7 +45,7 @@ func (e *listenerCommon) heartbeat() {
 
 	defer e.Close()
 
-	pingInterval := time.Second * 5
+	pingInterval := time.Second * 10
```
Comment (tomponline) [inline lxd/events/common.go:48]: Out of interest why was this needed?

File: lxc/console.go [hunk 2]
```diff
@@ -339,6 +339,12 @@ func (c *cmdConsole) vga(d lxd.InstanceServer, name string) error {
 		cmd.Stderr = os.Stderr
 		cmd.Start()
 
+		// Handle the command exitting.
```
Comment (tomponline) [inline lxc/console.go:342]: Typo

File: lxd/events/common.go (hunk 1 above)
Comment (stgraber) [reply lxd/events/common.go:48]: I've seen quite a few cases now where we seem to be hitting the timeout during initial forwarding through a busy LXD cluster.

Comment (tomponline) [reply lxd/events/common.go:48]: interesting, thanks

File: lxc/console.go (hunk 2 above)
Comment (stgraber) [reply lxc/console.go:342]: I'll put the typo fix in my master branch so it goes up with whatever next bugfix I send :)
